onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1214,6 +1214,62 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
 
+constexpr const char* GemmaRotaryEmbedding_ver1_doc = R"DOC(
+GemmaRotaryEmbedding is the implementation of below part of rotary positional embeddings (RoPE). It implements below from modeling_gemma.py.
+
+def gemma_rotary_embedding(emb, q, q_rot, k, k_rot):
+  sin_val = Sin(emb)
+  casted_sin = Cast(sin_val, to=fp16) # for fp16 mix-precision training. Other types are not supported.
+  cos_val = Cos(emb)
+  casted_cos = Cast(cos_val, to=fp16)
+  unsqueezed_sin = Unsqueeze(casted_sin)
```

Comment:
I assume Unsqueeze has two inputs. Where is the 2nd one?

Reply:
Current implementation checks and assumes the dimensions of input so unsqueeze_axis is not necessary. Should I add unsqueeze_axis as input?